import time

try:
    time.sleep()
except as exception:
    print("oh no! " + exception.arg)

print("Back from try/except")
